dit/dit.py

```diff
@@ -904,6 +916,19 @@ def raise_unrecognized_argument(argv):
             raise ArgumentException("Unrecognized argument: %s" % argv[0])
 
     # ===========================================
+    # Hooks
+
+    def _call_hook(self, hook):
+        if HOOKS_ENABLED:
+            hook_fn = self._hook_path(hook)
+            if os.path.isfile(hook_fn):
+                msg_normal("Executing hook: %s" % hook)
+                try:
+                    subprocess.run([hook_fn, self.base_path], check=CHECK_HOOKS)
```

Comment:
Capture the stdout and print it here in the main process. If the child processes are able to write to the stdout, the output will be out of sync.

```
cp = subprocess.run([hook_fn, self.base_path], check=CHECK_HOOKS, stdout=PIPE, universal_newlines=True)
sys.stdout.write(cp.stdout)
```